diff --git a/CONFIG b/CONFIG
index 6d5f0f17..217471f7 100644
--- a/CONFIG
+++ b/CONFIG
@@ -28,7 +28,6 @@ USE_GF2N_LONG = 1
 # ADX is used to optimize big integer additions
 # delete the second line to compile for a platform that supports everything
 ARCH = -mtune=native -msse4.1 -msse4.2 -maes -mpclmul -mavx -mavx2 -mbmi2 -madx
-ARCH = -march=native

 MACHINE := $(shell uname -m)
 ARM := $(shell uname -m | grep x86; echo $$?)
